backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveOrCopyDiskCommand.java

```diff
@@ -724,10 +714,29 @@ protected MultipleDiskVmElementValidator createMultipleDiskVmElementValidator()
                 diskVmElementDao.getAllDiskVmElementsByDiskId(getParameters().getImageGroupID()));
     }
 
-    public MultipleStorageDomainsValidator createMultipleStorageDomainsValidator(List<Guid> sdsToValidate) {
+    protected MultipleStorageDomainsValidator createMultipleStorageDomainsValidator() {
+        List<Guid> sdsToValidate = new ArrayList<>();
+        sdsToValidate.add(getStorageDomainId());
```

Comment:
so why do we need *multiple* storage domains validator rather than a single domain validator that validates the destination storage domain?

Reply:
For LSM another (overridden) `createMultipleStorageDomainsValidator()` is called which creates the `MultipleStorageDomainsValidator` with both source & destination Storage Domains' IDs (vs. only a destination Storage Domain ID in non-LSM case).
This allows the same code to be executed in `MoveOrCopyDiskCommand::validateSpaceRequirements()` method (i.e., `allDomainsWithinThresholds()`& `targetDomainHasSpaceForDisks()`) for 1 or more Storage Domains.
There is still not optimal, see another comment in the same commit:
https://github.com/oVirt/ovirt-engine/pull/624/commits/a792597418a4a934eeb38c6cd5e14f765b533743#r959582445

Reply:
ack